newIDE/app/src/ProjectManager/ProjectPropertiesDialog.js

```diff
@@ -417,6 +422,18 @@ class ProjectPropertiesDialog extends React.Component<Props, State> {
             value={adMobAppId}
             onChange={value => this.setState({ adMobAppId: value })}
           />
+          <Text size="title">
+            <Trans>Firebase</Trans>
+          </Text>
+          <SemiControlledTextField
+            floatingLabelText={<Trans>Firebase Configuration string</Trans>}
+            fullWidth
+            rows={8}
+            multiLine
```

Comment:
btw on the official material-ui multiLine is actually completely lowercase, and here the L is uppercase. That's an inconsistency and made me wonder for half an hour why it wasn't working until Bouh helped me by telling me to make the L Uppercase

Reply:
It's because historically material-ui v0 used `multiLine`, so when converting to Material-UI v4 the props name were kept - too much stuff to port already.

But feel free to submit a PR to rename this in the whole codebase.